spec/index.html

```diff
@@ -1021,11 +1021,13 @@ <h2>RDF Datasets</h2>
       can avoid interoperability issues by not ascribing importance to
       the presence or absence of empty named graphs.</p>
 
-    <p>SPARQL 1.2 [[SPARQL12-QUERY]] uses 
+    <p>Version 1.2 of the SPARQL Query Language [[SPARQL12-QUERY]] uses 
```

Comment:
It's all SPARQL specs which is why it says SPARQL 1.2. 

The link is provided because that is the starting point. Arguably, the link could be removed.

Reply:
> The link is provided because that is the starting point. Arguably, the link could be removed.

It seems odd for [[SPARQL12-QUERY](https://w3c.github.io/sparql-query/spec/)] to be the starting point.

[[SPARQL12-CONCEPTS](https://w3c.github.io/sparql-concepts/spec/)] or *possibly* [[SPARQL12-NEW](https://w3c.github.io/sparql-new/spec/)] makes more sense to me, and not only because these are listed before [[SPARQL12-QUERY](https://w3c.github.io/sparql-query/spec/)] in the **[List of Documents](https://w3c.github.io/sparql-query/spec/#related)** (to which I cannot find a way to link directly, nor to auto-expand after load)

> It's all SPARQL specs which is why it says SPARQL 1.2.

It seems to me, this lends weight to linking to [[SPARQL12-CONCEPTS](https://w3c.github.io/sparql-concepts/spec/)] or *possibly* [[SPARQL12-NEW](https://w3c.github.io/sparql-new/spec/)] instead of [[SPARQL12-QUERY](https://w3c.github.io/sparql-query/spec/)].

Removing the link is indeed also an option.

Reply:
Does this work for you, @afs?
```suggestion
    <p>SPARQL Version 1.2 [[SPARQL12-CONCEPTS]] uses 
```

Reply:
Yes - though "version" not "Version".